components/playlist/media_detector_component_manager.cc

```diff
@@ -230,7 +230,12 @@ void MediaDetectorComponentManager::SetUseLocalScriptForTesting() {
   }
 
   function getOGTagImage() {
-    return document.querySelector('meta[property="og:image"]')?.content
+    const url = document.querySelector('meta[property="og:image"]')?.content;
+    if (url && !url.startsWith('http')) {
```

Comment:
this is unlikely but a path like `httpsomething` would fail this check. you probably want to do `url && !url.startsWith('http://') && !url.startsWith('https://')` to be more precise

Reply:
Good point! Thanks. Done.